design/ceph/object-store.md

```diff
@@ -95,51 +92,16 @@ The RGW service can be configured to listen on both http and https by specifying
+- `zonegroup`: The name of the zonegroup the object store belongs in, if [Multisite Replication](/design/multisite-object-store.md) is enabled
+- `pull_endpoint`: The endpoint to pull the realm from if [Multisite Replication](/design/multisite-object-store.md) is enabled. If you are creating this zone on a different cluster than where the object store realm was created, then a resolvable endpoint to an RGW in the master zone for `pull_endpoint` is required. 
 
-By implementing on the independent realms, zone groups, and zones, Rook supports multiple objects stores in a cluster. The set of users with access to the object store, the metadata, and the data are isolated from other object stores.
-
-If desired to configure the object store to replicate from another cluster or zone, the following settings would be specified on a new object store that is *not* the master. (This feature is not yet implemented.)
-- `realm`: If specified, the new zone will be created in the existing realm with that name
-- `group`: If specified, the new zone will be created in the existing zone group with that name
-- `master`: If specified, settings indicate the RGW endpoint where this object store will need to connect to the master zone in order to initialize the replication. The Rook operator will execute `pull` commands for the realm and zone group as necessary.
 ```yaml
-  zone:
-    realm: myrealm
-    group: mygroup
-    master:
-      url: https://my-master-zone-gateway:443/
-      accessKey: my-master-zone-access-key
-      secret: my-master-zone-secret
-```
-
-Failing over the master could be handled by updating the affected object store CRDs, although more design is needed here.
-
-See the ceph docs [here](http://docs.ceph.com/docs/master/radosgw/multisite/) on the concepts around zones and replicating between zones. For reference, a diagram of two zones working across different cluster can be found on page 5 of [this doc](http://ceph.com/wp-content/uploads/2017/01/Understanding-a-Multi-Site-Ceph-Gateway-Installation-170119.pdf).
-
-
-### Ceph multi-site object store data model
-
-For reference, here is a description of the underlying Ceph data model.
-
-```
-A cluster has one or more realms
-
-A realm spans one or more clusters
-A realm has one or more zone groups
-A realm has one master zone group
-A realm defined in another cluster is replicated with the pull command
-
-A zone group has one or more zones
-A zone group has one master zone
-A zone group spans one or more clusters
-A zone group defined in another cluster is replicated with the pull command
-A zone group defines a namespace for object IDs unique across its zones
-Zone group metadata is replicated to other zone groups in the realm
-
-A zone belongs to one cluster
-A zone has a set of pools that store the user and object metadata and object data
-Zone data and metadata is replicated to other zones in the zone group
+  realm:
```

Comment:
Here we use "realm" but in the "object-store-realm.yaml" there is a "bootstrap-realm"... what is the difference?